src/unix/openbsd.c

```diff
@@ -190,12 +190,12 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   uv_cpu_info_t* cpu_info;
 
   size = sizeof(model);
-  if (sysctl(which, 2, &model, &size, NULL, 0))
+  if (sysctl(which, ARRAY_SIZE(which), &model, &size, NULL, 0))
```

Comment:
Not sure this is ok. Currently we pass 2 elements sometimes and other times we pass 3.

Reply:
I did not notice but normally the 0 ending value is not necessary.

Reply:
The third element with KERN_CPTIME2 is the CPU number, isn't it?

Reply:
Yes I expected some hits and misses in this change :-)